exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -138,3 +132,67 @@ impl CommandExtension for Finalize {
         Ok(context)
     }
 }
+
+pub struct Run;
+
+impl CommandExtension for Run {
+    fn args(&self) -> Vec<Argument> {
+        vec![
+            Argument::new_named(
+                EJB_JVM_ARGUMENTS,
+                false,
+                "Additional parameters for JVM. Must not have a leading dash. \
+                 For example, `Xmx2G` or `Xdebug`",
+                None,
+                "ejb-jvm-args",
+                true,
+            ),
+            Argument::new_named(
+                EJB_JVM_DEBUG_SOCKET,
+                false,
+                "Allows JVM being remotely debugged. Takes a socket address as a parameter in form \
```

Comment:
> Must not have a leading dash.

is redundant here (people don't usually start addresses with dashes unlike arguments `-127.1:5634`).